Dockerfile

```diff
@@ -9,6 +9,4 @@ ADD ./app ./app
 ADD ./main.sh ./main.sh
 
 # set up entrypoint
-CMD ["./main.sh"]
-ARG PORT=8080
-EXPOSE $PORT
```

Comment:
Why not expose the port?

Reply:
port information is in the k8s config now.

Reply:
Kubernetes is one way to deploy this application, but I don't think it should be the only one. Does the EXPOSE directive cause problems for k8s?

Reply:
i dont think so. k8s/helm will probably override it at deployment time. 

my goal was to remove clutter and insure that it works in linux and windows.